scalableminds/webknossos
# Prevent deleting teams that are still referenced

Fixed a bug where the user could delete teams that were still referenced in annotations, projects or task types, thus creating invalid state.

Also note follow-up issue #5109

### URL of deployed dev instance (used for testing):
- https://teamsdeleteassertion.webknossos.xyz

### Steps to test:
- Try deleting organization team, should be forbidden
- Try deleting team while not admin, should be forbidden
- Try deleting fresh team, should work
- Create team, create task type or project that uses that team, deleting it should now be forbidden.

### Issues:
- fixes #5077 

------
